eng/pipelines/templates/variables/globals.yml

```diff
@@ -8,4 +8,10 @@ variables:
   ${{ if ne(variables['System.TeamProject'], 'internal') }}:
     BuildConfiguration: 'Debug'
   ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-    BuildConfiguration: 'Release'
\ No newline at end of file
+    BuildConfiguration: 'Release'
+  # Publish to https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net
+  ${{ if eq(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net') }}:
+    DevOpsFeedID: '29ec6040-b234-4e31-b139-33dc4287b756/fa8c16a3-dbe0-4de2-a297-03065ec1ba3f'
+  # Publish to https://dev.azure.com/azure-sdk/internal/_packaging?_a=feed&feed=azure-sdk-for-net-pr
+  ${{ if eq(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net-pr') }}:
+    DevOpsFeedID: '590cfd2a-581c-4dcb-a12e-6568ce786175/fa8b2d77-74d9-48d7-bb96-badb2b9c6ca4'
```

Comment:
Just for my reference where do these id's come from? Are they part of a service connection or something else?

Reply:
The only way I could get the IDs was to add the NuGet task using the pipelines UI reference the feed then view the yaml generated. The actual ID does not seem to show up anywhere else.